packages/commons/package.json

```diff
@@ -59,4 +44,4 @@
   "dependencies": {
     "@types/aws-lambda": "^8.10.72"
```

Comment:
This can be moved as a dev-dependency too right?

Reply:
I need to check this one, I assumed it had to stay but maybe it can be removed. I'll test and get back to you

Reply:
Ok I've tested this and you were right, I removed it from this file and added it to the main `package.json` as `devDependency` in the last commit.

Reply:
That's great to hear, and it means the library's prod bundle decreased :)